src/components/flatSelect/FlatSelectOption.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { Content, IContentProps } from '../content/Content';
+import { ITooltipProps, Tooltip } from '../tooltip/Tooltip';
+
+export interface IFlatSelectOptionProps {
+  option: IContentProps;
+  classes?: string[];
+  prepend?: IContentProps;
+  append?: IContentProps;
+  tooltip?: ITooltipProps;
+  selected?: boolean;
+  onClick?: (option: IFlatSelectOptionProps) => void;
+}
```

Comment:
no `id` or `value`? You get the value by the option which is the content?

Reply:
I add an id for the flatSelectOption